unasync.py

```diff
@@ -36,7 +36,7 @@ def unasync_line(line):
 
 def unasync_file(in_path, out_path):
     with open(in_path, "r") as in_file:
-        with open(out_path, "w") as out_file:
+        with open(out_path, "w", newline="") as out_file:
```

Comment:
Could you clarify what this change is about?

Reply:
I am working on Windows, I have to set `newline=""` to prevent `universal newlines`, which auto converts "\n" to "\r\n" on Windows:

https://docs.python.org/3/library/functions.html#open
> When writing output to the stream, if newline is None, any '\n' characters written are translated to the system default line separator, os.linesep. If newline is '' or '\n', no translation takes place. If newline is any of the other legal values, any '\n' characters written are translated to the given string.

Reply:
Sounds good. Could we try and submit that as a separate pull request, for traceability purposes? (I might be maniac sometimes on clearly separating unrelated changes in separate PRs, but experience has proven it's beneficial when doing bug forensics…)

Reply:
Sure 👍

Reply:
Splited into #157